resources/secretsmanager_secrets.go

```diff
@@ -164,7 +164,7 @@ func fetchSecretsmanagerSecrets(ctx context.Context, meta schema.ClientMeta, _ *
 			}
 
 			secret := WrappedSecret{
-				SecretListEntry:   &n,
```

Comment:
Not sure if it'll work as `range` re-uses the value. Adding a `n := n` before, or accessing the element with `response.SecretList[i]` would be safer.